htdocs/core/modules/mrp/doc/pdf_vinci.modules.php

```diff
@@ -517,9 +517,9 @@ public function write_file($object, $outputlangs = '', $srctemplatepath = '', $h
 
 					if ($this->getColumnStatus('desc')) {
 						$pdf->startTransaction(); //description
-						$des = $prod -> description;
-						$descr = $des;//implode("<br>", $des);
-
+						/** $des = $prod -> description; it's far more logical to have the product's label instead of product's description
+						$descr = $des;//implode("<br>", $des); **/
+						$descr = $prod->label;
```

Comment:
So we should also change the key into getColumnStatus and into $this->cols['desc']

Reply:
I checked in some other modules (facture, order, propal), this col is named "desc", and it contains the product label (or the description if it's a free line)
So do you think it's realy necessary to change the col code from desc to label ?

Reply:
@eldy As we say in french "le mieux est l'ennemi du bien", and in this situation I really think it's better not to change the key in getColumnStatus 

- to keep this template similar to the others (facture, order, propal)
- to avoid problems to those have used the defineColumnField hook with desc key
- because changing this key could create some bugs